examples/jenkins/jenkins-persistent-template.json

```diff
@@ -303,6 +307,12 @@
       "displayName": "Jenkins ImageStreamTag",
       "description": "Name of the ImageStreamTag to be used for the Jenkins image.",
       "value": "jenkins:latest"
```

Comment:
how about here

Reply:
should be changed too, thanks for catching it.

Reply:
fixed.